### PR TITLE
feat(base-cluster/docs): add info about components

### LINKED ISSUE #290: [base-cluster] add more info in the docs
information about

- ingress
- cert-manager clusterIssuers

### PR DESCRIPTION
**Pull request by cwrau**
Closes #290

### CODE REVIEW

**Comment by cwrau** [comment]
@mw do you think something is missing?

File: charts/base-cluster/README.md.gotmpl [hunk 1]
```diff
@@ -37,24 +37,153 @@ git push
...
+### Component [descheduler](#descheduler)
+
+The [descheduler](https://github.com/kubernetes-sigs/descheduler) runs periodically
+and tries to average the load across the nodes by deleting pods on fuller nodes
+so the kube-scheduler can, hopefully, schedule them on nodes with more space.
+
+Additionally, the descheduler also tries to reconcile `topologySpreadConstraints`
+and affinities.
+
+If the cluster is _semi_ underspecced or the individual applications have unperfect
```
Comment (marvinWolff) [inline charts/base-cluster/README.md.gotmpl:76]: This is a nice phrase :D

File: charts/base-cluster/README.md.gotmpl [hunk 2]
```diff
@@ -37,24 +37,153 @@ git push
...
+      you can explicitly choose the Issuer by using these annotations:
+      - `cert-manager.io/cluster-issuer: letsencrypt-staging`
+      - `cert-manager.io/cluster-issuer: letsencrypt-production`
+
+#### IP Address
+
+If you want to make sure that, in the event of a catastrophic failure, you keep the
+same IP adress, you should roll this out, get the assigned IP
+(`kubectl -n ingress-nginx get svc ingress-nginx-controller -o jsonpath='{.status.loadBalancer.ingress}'`)
+and set `.ingress.IP=<ip>`. This makes sure the IP is kept in your project (may incur cost!)
```
Comment (marvinWolff) [inline charts/base-cluster/README.md.gotmpl:111]: In the `values.yaml`?
Maybe clarify that.

File: charts/base-cluster/README.md.gotmpl [hunk 3]
```diff
@@ -37,24 +37,153 @@ git push
...
+for [Horizontal Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale),
+[Vertical Autoscaling](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler)
+and to allow `kubectl top` and tools like [k9s](https://k9scli.io) to show
+resource usage for your pods and nodes.
+
+#### Sub-Component [securityScanning](#monitoring_securityScanning)
+
+The included [trivy](https://aquasecurity.github.io/trivy-operator) scans the
+running workload in your cluster for CVEs and creates
+[Custom Resources](https://aquasecurity.github.io/trivy-operator/v0.12.1/docs/crds).
```
Comment (marvinWolff) [inline charts/base-cluster/README.md.gotmpl:160]: Maybe add a sentence that you can also monitor the found CVEs in your cluster.
Somethin like:

```
... creates Custom Resources to monitor the results.
```
or something like that.

**Pull request merged**